src/pages/cli/start/install.mdx

```diff
@@ -23,11 +21,86 @@ import all0 from "/src/fragments/cli-install-block.mdx";
 
 To set up the Amplify CLI on your local machine, you have to configure it to connect to your AWS account.
 
+> If you already have an AWS profile with credentials on your machine, you can skip this step.
+
+> If you are using CLI version <= 10.6.2 go to [Configure CLI <= 10.6.2](#configure-the-amplify-cli-version--1062)
+
+Configure Amplify by running the following command:
+
+```bash
+amplify configure
+```
+
+`amplify configure` will ask you to sign into the AWS Console.
+
+Once you're signed in, Amplify CLI will ask you to create an IAM user.
+
+> Amazon IAM (Identity and Access Management) enables you to manage users and user permissions in AWS. You can learn more about Amazon IAM [here](https://aws.amazon.com/iam/).
+
+```console
+Specify the AWS Region
+? region:  # Your preferred region
+Follow the instructions at
+https://docs.amplify.aws/cli/start/install/#configure-the-amplify-cli
+
+to complete the user creation in the AWS console
+https://console.aws.amazon.com/iamv2/home#/users/create
+```
+
+Navigate to the [IAM User creation page](https://console.aws.amazon.com/iamv2/home#/users/create) if it's not already open.
+
+Enter a User name and click Next. You can name the user anything but we'll call it "amplify-dev".
```

Comment:
```suggestion
Enter a User name and select **Next**. You can name the user anything but we'll call it "amplify-dev".
```

minor adjustment for styleguide https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md#15-use-carets-to-help-users-navigate-menus-use-select-instead-of-click